apps/testrunner/test_suites/selenium_tests/selenium_timer.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+class SeleniumTimer(object):
+
+    def __init__(self, driver):
+        self.driver = driver
+        self.measurements = []
+
+    def start(self):
+        self.start_timestamp = self.driver.execute_script("return new Date().getTime()")
+
+    def mark_measurement_point(self, url):
+        timing = self.driver.execute_script("return window.performance.timing")
+
+        self.measurements.append(dict(
+            url = url,
+            backend_time = timing['responseStart'] - timing['navigationStart'],
+            frontendTime = timing['loadEventEnd'] - timing['responseStart'],
+            totalLoadTime = timing['loadEventEnd'] - timing['navigationStart'],
+            response_receiving_time = timing['responseEnd'] - timing['responseStart'],
+            interactive_time = timing['domInteractive'] - timing['navigationStart'],
+            dom_complete_time = timing['domComplete'] - timing['navigationStart'],
+            dom_content_loaded_time = timing['domContentLoadedEventEnd'] - timing['navigationStart']
+        ))
+
+        self.total_load_time = timing['loadEventEnd'] - self.start_timestamp
+
+    def get_result(self):
+        if len(self.measurements) == 0:
+            return None
```

Comment:
Perhaps it depends on the context. If you have it stashed somewhere in one block of try-except like this:
```python
try:
    ret1 = fun1()
    ret2 = fun2(ret2)
    print(ret2)
except Error as ex:
    log.error("Some error happened: {0}".format(ex))
```
it's better then checking errors manualy
```python
ret1 = fun1()
if ret1 is None:
    log.error("Some error happened")
    return None

ret2 = fun2(ret1)
if ret2 is None:
    log.error("Some other error happened")
    return None
print(ret2)
```
take a look at Dict.get() for example https://docs.python.org/2/library/stdtypes.html#mapping-types-dict.

On the other hand shouldn't we have an property instead of getter? More pythonic way :)